examples/active.ini

```diff
@@ -30,7 +31,7 @@ num_t1s = 2
 uw_dip_center_ghz = 9.820639
 uw_dip_width_ghz = 0.02
 fir_rep = 500000.0
-min_dBm_step = 1.0
+min_dbm_step = 1.0
```

Comment:
IDK if this should be a lower or uppercase 'B'

Reply:
I see -- I don't know if it matters, but switching to lowercase if fine